rust/repo/{{ cookiecutter.repo_name }}/.github/labeler.yaml

```diff
@@ -4,10 +4,12 @@ rust:
   - '**/Cargo.lock'
   - '**/Cargo.toml'
 
-dependencies:
-  - '**/Cargo.lock'
-
 github_actions:
   - '.github/workflows/**'
   - '.github/labeler.yaml'
   - '.github/triage-labeler.yaml'
+
+python:
+  - '**.py'
+  - '**/pyproject.toml'
+  - '**/poetry.lock'
```

Comment:
:question: In the labeler for the cookiecutters I said don't cross the streams.
But here I'm wondering if we want to be pushing new projects to poetry or if should this be more general and allow requirements.txt?  

I feel like poetry or similar should be the way to go, but I know it can also take one a little bit to understand the whole venv thing in python. While some people use a requirements.txt file to get the "system" python set up.

Reply:
I think we should do both. If we see a `requirements.txt`, it's obviously python-related, so we can label it as such.

But we should also push projects to use poetry.